CefSharp/BindingOptions.cs

```diff
@@ -29,5 +29,12 @@ public static BindingOptions DefaultBinder
         /// for logging calls (from js) to .net methods.
         /// </summary>
         public IMethodInterceptor MethodInterceptor { get; set; }
+
+        /// <summary>
+        /// Interceptor used for intercepting get/set calls to the target object property. For instance, can be used 
+        /// for logging calls to .net property (from js)
+        /// </summary>
+        public IPropertyInterceptor PropertyInterceptor { get; set; }
```

Comment:
Should be excluded from `netcore` project, shouldbe something like the following. Same will apply to other references.

```c#
#if !NETCOREAPP
        /// <summary>
        /// Interceptor used for intercepting get/set calls to the target object property. For instance, can be used 
        /// for logging calls to .net property (from js)
        /// </summary>
        public IPropertyInterceptor PropertyInterceptor { get; set; }
#endif
```

Reply:
@amaitland Comment incorporated.